app/styles/app.scss

```diff
@@ -0,0 +1,2 @@
+
+@import "ember-power-select";
```

Comment:
I took back what I said earlier since there should be a fix for this now.  You shouldn't have to do the `@import "ember-power-select"` with the newest version of ember-osf.